src/header/accept.rs

```diff
@@ -54,7 +54,7 @@ impl WebSocketAccept {
 		let mut concat_key = String::with_capacity(serialized.len() + 36);
 		concat_key.push_str(&serialized[..]);
 		concat_key.push_str(MAGIC_GUID);
-		let output = hash(hash::Type::SHA1, concat_key.as_bytes());
+		let output = hash(hash::MessageDigest::sha1(), concat_key.as_bytes()).unwrap();
```

Comment:
hash now outputs a result with an ssl errorstack. hashing can fail at a number of steps where openssl calls into c. if there's an error, we're just going to want to fail.